More differential treatment (#147)
ietf-wg-ohai/oblivious-http: draft-ietf-ohai-ohttp.md

A relay MAY add information to requests if the client is aware of the nature of
the information that could be added.  The client does not need to be aware of
the exact value added for each request, but needs to know the range of possible
values the relay might use.  Importantly, information added by the relay - beyond

> what is already revealed through encapsulated requests from clients - can reduce

Reviewer: ```suggestion
what is already revealed in the content of a request - can reduce
```

Author: I think we ought to keep this as is, since the motivating reason for this change was that the config used to encapsulate a request can partition the client anonymity set.

Some patterns of abuse cannot be detected without access to the request that
is made to the target. This means that only the gateway or target are in a
position to identify abuse. A gateway MAY send signals toward the relay to
provide feedback about specific requests. For example, a gateway could respond
differently to requests it cannot decapsulate, as mentioned in {{errors}}. A
relay that acts on this feedback could - either inadvertently or by

> design - lead to client deaononymization.

Reviewer 2: ```suggestion
design - lead to client deanonymization.
```
